ui/components/GrafanaCustomChart.js

```diff
@@ -493,7 +493,7 @@ class GrafanaCustomChart extends Component {
       }
     }
 
-    transformDataForChart(data, target) {
```

Comment:
Why remove the target? See https://github.com/layer5io/meshery/blob/a9ee4fd229c54c2de7dc0699dcd8501e8d0f2264/ui/components/GrafanaCustomChart.js#L409

Reply:
I think you need to take a better look at code again, giving a unused argument to function is not a good way to write code.